cat/opencl.cat

```diff
@@ -92,8 +98,8 @@ irreflexive rf; (ghb | lhb) as O-Rf
 let vis(hb) = (W * R) & hb & loc & ~((hb & loc); [W]; hb)
 
 (* A non-atomic load can only read from a store that is visible. *)
-empty (rf; [NAL & GLOBAL])\vis(ghb) as O-NaRfG
-empty (rf; [NAL & LOCAL])\vis(lhb) as O-NaRfL
+empty (rf; dynamic_NAL & dynamic_GLOBAL)\vis(ghb) as O-NaRfG
+empty (rf; dynamic_NAL & dynamic_LOCAL)\vis(lhb) as O-NaRfL
```

Comment:
I don't remember what is the precedence of `;` wrt `&`, but better to use `()` here

Reply:
Added